playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -13,6 +20,7 @@
     autorefresh: True
     auto_import_keys: yes
     state: present
+  when: not product_present
```

Comment:
cant we use the command here directly?

Reply:
I don't know, can we? I still feel new to ansible...

Reply:
@evrardjp might be the person to ask this :D

Reply:
I think one can't execute code in `when`